lib/components/primitive-components/Group/Group_doInitialPcbLayoutPack.ts

```diff
@@ -117,6 +118,15 @@ export const Group_doInitialPcbLayoutPack = (group: Group) => {
             "pcb_component_id" in elm && elm.pcb_component_id === componentId,
         )
       transformPCBElements(related as any, transformMatrix)
+      const groupRot = group._parsedProps?.pcbRotation ?? 0
+      const packedRot = ccwRotationDegrees ?? ccwRotationOffset ?? 0
+      const existingRot = pcbComponent.rotation ?? 0
+      const finalRot =
+        (((groupRot + packedRot + existingRot) % 360) + 360) % 360
+
+      db.pcb_component.update(componentId, {
+        rotation: finalRot,
+      })
```

Comment:
this is good, but i believe this should be handled inside the circuit-json-util module no?

also isn't the issue related to schematic not pcb?